### PR TITLE
Added ALLOWED_EXPORT_PATHS setting

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #6335

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: pulpcore/app/models/exporter.py [hunk 1]
```diff
@@ -100,7 +100,13 @@ def _export_to_file_system(self, content_artifacts):
 
         Args:
             content_artifacts (django.db.models.QuerySet): Set of ContentArtifacts to export
+
+        Raises:
+            ValidationError: When path is not in the ALLOWED_EXPORT_PATHS setting
         """
+        from pulpcore.app.serializers import ExportSerializer
```
Comment (bmbouter) [inline pulpcore/app/models/exporter.py]: Circular import avoided?

Comment (bmbouter) [reply pulpcore/app/models/exporter.py]: I want to confirm, but we have this issue in several places so we shouldn't solve with this piece of work here.

Comment (daviddavis) [reply pulpcore/app/models/exporter.py]: Haha yes. I remember [the discussion we had on the import PR where I suggested we should leave a comment](https://github.com/pulp/pulpcore/pull/520#discussion_r372606433). I can do that if it's not clear.

File: pulpcore/app/serializers/exporter.py [hunk 2]
```diff
@@ -24,6 +25,25 @@ class ExporterSerializer(ModelSerializer):
...
+            value: The user-provided value path to be validated.
+
+        Raises:
+            ValidationError: When path is not in the ALLOWED_EXPORT_PATHS setting.
+
+        Returns:
+            The validated value.
+        """
+        for allowed_path in settings.ALLOWED_EXPORT_PATHS:
+            user_provided_realpath = os.path.realpath(value)
```
Comment (bmbouter) [inline pulpcore/app/serializers/exporter.py]: :+1:

File: pulpcore/app/models/exporter.py (hunk 1 above)
Comment (bmbouter) [reply pulpcore/app/models/exporter.py]: nope, this is fine.

**Pull request merged**